core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -204,7 +204,9 @@
   }
   .oppia-review-right-details-container {
     left: 40%;
+    max-width: 300px;
     position: relative;
+    width: 40%;
```

Comment:
These two changes are not related to the original issue. They have been made because of the following reason:

Originally, without these two lines, the dashboard looked normal on wider screens. However, when the screen width fell below ~900px, the div containing the 'Review rights' info refused to shrink, causing it to flow out of its parent div:
![image](https://user-images.githubusercontent.com/78612244/143925679-d321ae14-adc9-4e7e-a167-136dd802d92d.png)

These two lines fix the issue.
 - The width property allows it to shrink on screens with smaller widths.
 - The max-width property stops it from growing when the screen is sufficiently wide.